lib/querystring.js

```diff
@@ -158,6 +172,10 @@ function qsEscape(str) {
   return encodeStr(str, noEscape, hexTable);
 }
 
+/**
+ * @param {string | number | BigInt | boolean} v
```

Comment:
```suggestion
 * @param {string | number | bigint | boolean} v
```
Type `BigInt` is not recognized, `BigInt` -> `bigint`.

Reply:
Yep; `BigInt` is the Object wrapped form, typeof `bigint` is the primitive form. Odd that TS doesn't recognize the Object wrapped form but this needs to be changed either way.